Aptoide/aptoide-client-v8
# [Cobrand/ASV-1207] Cobrand package name & version name.

**What does this PR do?**

   Adds cobrand store name in front of the package name. Example: cm.aptoide.pt.<store_name>
   Adds .c<version> in front of the version name. Example: "9.3.0.1.c1" 
   Extracts applicationId to the default configuration and uses it in different flavours.
**Database changed?**

   No

**Where should the reviewer start?**

- [ ] build.gradle (app)

**How should this be manually tested?**

  Build different vanilla/cobrand variants and check both package and version name.

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-1207](https://aptoide.atlassian.net/browse/ASV-1207)

**Code Review Checklist**
